### DISCUSSION TITLE
Improvment unlock button

### DISCUSSION THREAD

**Discussion by phildev4** [AdguardTeam/AdGuardHome#2869] [Q&A]
### Problem Description
It is not really a problem. However, I think it would be an enhancement of the user experience.

### Proposed Solution
When you click on unblock in the query log it will add the blocked website to the custom filtering list. However, in order to work, you have to click apply in the custom filtering tab. I think this should be done automatically.

**Comment by ameshkov** [comment]
Probably, this is a timing issue. It should be applied right after the click, but it may take some time to re-compile the filtering engine.
